Marlin/Configuration.h

```diff
@@ -574,6 +576,12 @@
 #define PREVENT_LENGTHY_EXTRUDE
 #define EXTRUDE_MAXLENGTH 200
 
+// @section CHAMBER
```

Comment:
THIS IS RELEVANT FOR THE PULL REQUEST
adds the option to enable chamber fan and chamber vent

Reply:
You can't overwrite the default configs here with those for a specific printer. Just add changes relevant to the PR/feature and revert the rest.

You can submit a printer-specific config to the [Marlin Configurations repo](https://github.com/MarlinFirmware/Configurations/).